src/containers/Application/ApplicationPageWrapper.tsx

```diff
@@ -79,7 +79,7 @@ const ApplicationPageWrapper: React.FC = () => {
   // 1 - ProcessRedirect: Will redirect to summary in case application is SUBMITTED
   // 2 - Set the current section state of the application
   useEffect(() => {
-    if (isApplicationLoaded) {
+    if (elementsState && responsesByCode) {
       processRedirect({
```

Comment:
Don't need to use await here?

Reply:
I don't think so, cos if there is a redirect required, then the called method does the `push`, so it'll over-rule anything else in progress. And if there's no redirect, then we don't need to wait for anything.

Reply:
Ah! Got it. That's good.